admin/acceptance_tests/metadatas_test.py

```diff
@@ -184,6 +184,58 @@ def test_valid_int_metadata(self, test_app, metadatas_test_data):
             '2',
             302)
 
+    def test_invalid_url_metadata(self, test_app, metadatas_test_data):
+        self._post_invalid_metadata(
+            test_app,
+            '/layers_wms/new',
+            self._base_metadata_params(metadatas_test_data),
+            '_url',
+            'gnagnagna',
+            'Must be a URL')
```

Comment:
an URL

Reply:
So says colander.url validator ! Do we have to translate it ?

Reply:
Seems already done in colander : https://github.com/Pylons/colander/blob/master/colander/locale/fr/LC_MESSAGES/colander.po#L59
If not done we should add colander translations in our translation fallback system in related story.